design/token-mechanics-v2.1.md

```diff
@@ -1,46 +1,55 @@
-- Challengers will challenge a specific version of a package.
-- The minimum stake is validated against the summation of the amount vouched by the package owner for each of its version.
-- For non-owner users, there is no penalization for unvouching/moving tokens. Package owners will have to make sure the minimum stake is reached.
-- Packages reputation could be calculated based on total vouch amount weighted by the registered versions
-
-### Issues
-1- How we can avoid users unvouching their tokens given a new critical challenge yet unsolved?
-> We can either use cyclic windows to state when users can challenge, when users can vouch/unvouch, and when challenges 
-  will be solved. Or, we can lock a fraction of the vouched tokens of each user to ensure we can pay successful 
-  challenges. Both options have their downsides: locking may not be possible if there is a large list of challenges 
-  if the payout formula depends on the total stake. OTOH, windows may no be the ideal solution for critical challenges
-
-2- Challenges front-running?
-> A simple way to solve this issue for the following version, is to can handle front-running scenarios manually checking both challenges and deciding 
-  which is the real one.
-
-3- Challenges payout formula
-  3.1- Severity
-  > Each challenge can have a severity label given by the challenger. The severity may affect the payout received by each challenger.
-
-  3.2- How we can make sure challenges payout cannot be manipulated based on their resolution order? 
-  > An obvious approach would be to solve challenges in a FIFO order. Note that this blocks solving a challenge until all previous ones have been 
-    solved as well. Therefore, if each challenge receives a percentage of the tokens staked, then the order in which challenges are resolved 
-    actually alters how much of a reward each challenge gets.
-
-  3.3- How we can make sure there is enough tokens to pay successful challenges?
-  > We should decide whether we want to accept defaults or not, i.e. do we want to consider an scenario where a package does not have enough 
-    tokens staked to pay a challenge? If not, we should pay a proportional amount of tokens based on the severity and the total stake.
-    
-  3.4- How we will handle a huge list of vouchers during a payout?
-  > We can have a rate per version in ZEP that will be adjust each time a challenge is solved.  
+# Token Mechanics v2.1
+
+## Protocol
+
+### Vouching
+
+- Developers will register plain addresses. These addresses may represent anything: an EVM package, an EVM package version, a logic contract from a package, a contract instance, or a mock address representing something off-chain. 
+- Registering a new entry requires a minimum stake that cannot be removed, and is used for spam prevention. The minimum stake can only be moved to another entry owned by the same developer, but never withdrawn from the vouching system.
```

Comment:
This means that an entry could be left without a minimum stake backing it, is that correct?